lts.json

```diff
@@ -2,13 +2,13 @@
   "autoprefixer": "~0.6.0",
   "broccoli-asset-rev": "^2.4.2",
```

Comment:
The final pinning of versions is the responsibility of `shrinkwrap` (we can't control the versions down the stack otherwise) - in these cases we aren't consuming the dependencies directly, but rather because we're consuming `ember-cli`, so really we only want to list the version of the top level dependency being used